libvast/src/detail/signal_handler.cpp

```diff
@@ -0,0 +1,28 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/detail/signal_handler.hpp"
+
+#include "vast/config.hpp"
+#include "vast/detail/backtrace.hpp"
+
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+
+extern "C" void signal_handler(int sig) {
```

Comment:
Should this function be marked `[[noreturn]]`? Is that even possible with `extern "C"`?

Reply:
I changed to implementation. It doesn't call `quick_exit` any more.